pkg/deployments/tasks/20230314-batch-relayer-v5/test/test.fork.ts

```diff
@@ -145,4 +152,39 @@ describeForkTest('BatchRelayerLibrary', 'mainnet', 15485000, function () {
 
     expect(await destinationGauge.balanceOf(sender.address)).to.be.gt(0);
   });
+
+  it('sender can update their gauge liquidity limits', async () => {
+    const tx = await relayer.connect(sender).multicall([
+      library.interface.encodeFunctionData('manageUserBalance', [
+        [RETH_STABLE_GAUGE, ETH_STETH_GAUGE].map((gauge) => ({
+          kind: TRANSFER_EXTERNAL_USER_BALANCE_OP_KIND,
+          asset: gauge,
+          amount: 1, // The amount must be non-zero to trigger the token trasnfer and liquidity limit update
+          sender: sender.address,
+          recipient: sender.address,
+        })),
+        0,
+      ]),
+    ]);
+
+    const gaugeInterface = new ethers.utils.Interface([
+      'event UpdateLiquidityLimit(address indexed user, uint256 original_balance, uint256 original_supply, uint256 working_balance, uint256 working_supply)',
+    ]);
+
+    expectEvent.inIndirectReceipt(
+      await tx.wait(),
+      gaugeInterface,
+      'UpdateLiquidityLimit',
+      { user: sender.address },
+      RETH_STABLE_GAUGE
+    );
+
+    expectEvent.inIndirectReceipt(
+      await tx.wait(),
+      gaugeInterface,
+      'UpdateLiquidityLimit',
+      { user: sender.address },
+      ETH_STETH_GAUGE
+    );
+  });
```

Comment:
Fair enough. I thought about checking the balance and / or the working supply before the update, but on second thought  it doesn't really add much value in this context.